README.md

```diff
@@ -11,42 +11,41 @@ Enjoy.
 
```

Comment:
I'd add name of the project (Luncher)and one-sentence description what this app is used for: searching recipes by ingredients; maybe we can also do what another group: put the link to User Stories?